From 06fe890db9b7798363b814269f2a6cc2ec594e8f Mon Sep 17 00:00:00 2001
From: Jon Ringle <jon@ringle.org>
Date: Thu, 25 Mar 2010 22:38:53 -0400
Subject: [PATCH] replace bzero with memset

Work on this was done in commit c0f19ccff73aa0ec5e9f3b61b932ea4a8bf4e6ed,
however a couple of bzero call sites were missed

Signed-off-by: Jon Ringle <jon@ringle.org>
---
 fdisk/fdiskbsdlabel.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdisk/fdiskbsdlabel.c b/fdisk/fdiskbsdlabel.c
index 2f41561..9c09252 100644
--- a/fdisk/fdiskbsdlabel.c
+++ b/fdisk/fdiskbsdlabel.c
@@ -540,7 +540,7 @@ xbsd_write_bootstrap (void)
   memmove (&dl, d, sizeof (struct xbsd_disklabel));
 
   /* The disklabel will be overwritten by 0's from bootxx anyway */
-  bzero (d, sizeof (struct xbsd_disklabel));
+  memset (d, 0, sizeof (struct xbsd_disklabel));
 
   snprintf (path, sizeof(path), "%s/boot%s", bootdir, dkbasename);
   if (!xbsd_get_bootstrap (path, &disklabelbuffer[xbsd_dlabel.d_secsize],
@@ -656,7 +656,7 @@ xbsd_initlabel (struct partition *p, struct xbsd_disklabel *d, int pindex) {
 	struct geom g;
 
 	get_geometry (fd, &g);
-	bzero (d, sizeof (struct xbsd_disklabel));
+	memset (d, 0, sizeof (struct xbsd_disklabel));
 
 	d -> d_magic = BSD_DISKMAGIC;
 
-- 
1.7.0.2.182.ge007

